configs/repos/ufs/packages/nceplibs-bundle/package.py

```diff
@@ -53,6 +53,6 @@ class NceplibsBundle(BundlePackage):
     depends_on('w3emc')
     depends_on('w3nco')
     depends_on('wrf-io')
-    depends_on('wgrib2')
+    #depends_on('wgrib2')
```

Comment:
```
  Fetch: 1.37s.  Build: 0.42s.  Total: 1.79s.
[+] /glade/work/heinzell/jedi-stack/spack-stack-develop-20220215/envs/cheyenne-intel-2021.2.0-all/install/intel/2021.2.0/shumlib-macos_clang_port-qtrwavv
==> Error: Unable to parse extension from https://www.ftp.cpc.ncep.noaa.gov/wd51we/wgrib2/wgrib2.tgz.v2.0.8-cmake-v6.

If this URL is for a tarball but does not include the file extension
in the name, you can explicitly declare it with the following syntax:

    version('1.2.3', 'hash', extension='tar.gz')

If this URL is for a download like a .jar or .whl that does not need
to be expanded, or an uncompressed installation script, you can tell
Spack not to expand it with the following syntax:

    version('1.2.3', 'hash', expand=False)
```

Reply:
This will be fixed with #32

Reply:
Ok, when #32 gets merged after this PR, let's put it back in.